core/services/base.py

```diff
@@ -72,3 +71,14 @@ def _adjust_update_payload(self, payload_data):
 
     def _base_payload_adjust(self, obj_data):
         return obj_data
+
+
+def wait_for_mutation(client_mutation_id):
+    mutation = MutationLog(client_mutation_id=client_mutation_id)
+    if not mutation:
+        return
+    loop_count = 0
+    while mutation.status == MutationLog.RECEIVED and loop_count>10:
```

Comment:
This is not correct and will never execute as loop_count>10 will first give False. 
You wanted to say: `while mutation.status == MutationLog.RECEIVED and loop_count<=10:` ?
Also, why 10? You wait for 3s? Should this be a constant or a module configuration? 
Please check and fix.

Reply:
no it is just a security, honestly this code should not be used, the FE should wait